### ISSUE TITLE
Installation failed: Conclusion: remove nikic/php-parser v4.10.2

### ISSUE THREAD

**Issue by 4n70w4** [infection/infection#1353]
```bash
composer require --dev infection/infection
```

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install infection/infection 0.18
    - Conclusion: remove nikic/php-parser v4.10.2
    - Installation request for infection/infection ^0.18.0 -> satisfiable by infection/infection[0.18, 0.18.x-dev].
    - Conclusion: don't install nikic/php-parser v4.10.2
    - infection/infection 0.18.x-dev requires nikic/php-parser ^4.9 <4.10 -> satisfiable by nikic/php-parser[4.9.x-dev, v4.9.0, v4.9.1].
    - Can only install one of: nikic/php-parser[4.9.x-dev, v4.10.2].
    - Can only install one of: nikic/php-parser[v4.9.0, v4.10.2].
    - Can only install one of: nikic/php-parser[v4.9.1, v4.10.2].
    - Installation request for nikic/php-parser (locked at v4.10.2) -> satisfiable by nikic/php-parser[v4.10.2].


Installation failed, reverting ./composer.json to its original content.
```

Workaround:

```bash
composer require nikic/php-parser:4.9
```

And retry:

```bash
composer require --dev infection/infection
```

**Comment by sanmai** [comment]
There's a problem with newer versions of PHP-Parser, which of I'm not yet aware of a fix or resolution.

**Comment by maks-rafalko** [comment]
Released as 0.18.2
